jdaviz/core/template_mixin.py

```diff
@@ -1281,7 +1285,26 @@ def _on_layers_changed(self, msg=None):
         viewers = [self._get_viewer(viewer) for viewer in viewer_names]
 
         manual_items = [{'label': label} for label in self.manual_options]
-        layers = [layer for viewer in viewers for layer in getattr(viewer, 'layers', [])]
+
+        # use getattr so the super() call above doesn't try to access the attr before
+        # it is initialized:
+        if not getattr(self, 'only_wcs_layers', False):
+            layers = [
+                layer for viewer in viewers
+                for layer in getattr(viewer, 'layers', [])
+                # don't include WCS-only layers unless asked:
+                if (
+                    not hasattr(layer.layer, 'meta') or
+                    (not layer.layer.meta.get('_WCS_ONLY', False))
+                )
+            ]
+        else:
+            layers = [
+                layer for viewer in viewers
+                for layer in getattr(viewer, 'layers', [])
+                # only include WCS-only layers:
+                if not hasattr(layer.layer, 'meta') or layer.layer.meta.get('_WCS_ONLY', False)
```

Comment:
```suggestion
                if (not hasattr(layer.layer, 'meta') or layer.layer.meta.get('_WCS_ONLY', False))
                and "Subset" not in layer.layer.label
```

This is the simplest fix for the Subsets appearing in the orientation menu.

Reply:
You win. 👏🏻 🎉